chain/pool/txpool.go

```diff
@@ -334,17 +336,6 @@ func (tp *TxnPool) verifyTransactionWithLedger(txn *Transaction) error {
 		return ErrDuplicatedTx
 	}
 
-	// get signature chain from commit transaction then add it to POR server
-	if txn.UnsignedTx.Payload.Type == pb.CommitType {
```

Comment:
Move the TypeCheck into switch ... case maybe cause many unnecessary `Unpack(txn.UnsignedTx.Payload)`

Reply:
The Unpack will be removed later when we merge some part of the verification with ledger